packages/flutter_driver/lib/src/driver/timeline.dart

```diff
@@ -25,106 +25,105 @@ class Timeline {
   ///
   /// This is parsed from "traceEvents" data within [json] and sorted by
   /// timestamp. Anything without a valid timestamp is put in the beginning.
-  final List<TimelineEvent> events;
+  final List<TimelineEvent>? events;
```

Comment:
It's surprising that this can be null. Can we document that? Or should it always be an (sometimes empty) list?

Reply:
Hmm. I'm pretty sure the method that parses this was written in an overly cautious way. I was going to not change it, but I'm realizing now that it can only return null if another thing thatisn't allowed to be null (`json`) was null. I'm going to change it.

Reply:
I was wrong.

This can be null if the trace events are null in the JSON. I'll document it for now.

Reply:
I think changing it to an empty list would be fine, but would like to do that separately from this migration in case anything depends on this behavior.